README.md

```diff
@@ -391,7 +391,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [delorean](https://github.com/myusuf3/delorean/) - A library for clearing up the inconvenient truths that arise dealing with datetimes.
 * [moment](https://github.com/zachwill/moment) - A Python library for dealing with dates/times. Inspired by [Moment.js](http://momentjs.com/).
 * [Pendulum](https://github.com/sdispater/pendulum) - Python datetimes made easy.
-* [PyTime](https://github.com/shinux/PyTime) - A easy-use Python module which aims to operate date/time/datetime by string.
+* [PyTime](https://github.com/shinux/PyTime) - An easy-to-use Python module which aims to operate date/time/datetime by string.
```

Comment:
Hello @koreissm,

The description of [PyTime's repository](https://github.com/shinux/PyTime) says:
> PyTime is an easy-use Python module which aims to operate date/time/datetime by string.

So it's worth to use `easy-use` instead of `easy-to-use` in order to be consistent.
```suggestion
* [PyTime](https://github.com/shinux/PyTime) - An easy-use Python module which aims to operate date/time/datetime by string.
```